shell/browser/file_select_helper_mac.mm

```diff
@@ -0,0 +1,136 @@
+// Copyright 2014 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
Are there any additional changes apart from the upstream implementation, can instead include `//chrome/browser/file_select_helper_mac.mm` ?

Reply:
@deepak1556 we can potentially - we'd need to patch though :(

```
In file included from ../../chrome/browser/file_select_helper_mac.mm:5:
In file included from ../../chrome/browser/file_select_helper.h:27:
In file included from ../../chrome/browser/safe_browsing/cloud_content_scanning/deep_scanning_dialog_delegate.h:19:
../../chrome/browser/enterprise/connectors/common.h:11:10: fatal error: 'components/enterprise/common/proto/connectors.pb.h' file not found
#include "components/enterprise/common/proto/connectors.pb.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

Reply:
It looks like we could set `BUILDFLAG(FULL_SAFE_BROWSING)` to false in order to omit that include...

essentially just `safe_browsing_mode = 0` in args.gn.